vocabularies/UI.xml

```diff
@@ -1549,6 +1549,14 @@ or `204 No Content` if the request included a `Prefer` header with a value of `r
         </Annotation>
       </Term>
 
+      <Term Name="IsAIOperation" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Action Function ActionImport FunctionImport">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The annotated operation leverages AI capabilities." />
```

Comment:
```suggestion
        <Annotation Term="Core.Description" String="The annotated operation is powered by AI" />
```

Reply:
sounds more like a marketing campaign, but would be OK for me.